api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/services/cached/CachedProductFamilyService.java

```diff
@@ -386,6 +389,8 @@ private boolean update(APIService apiService) {
         apiService.setStatus(isUp ? "UP" : "DOWN");
         apiService.setSsoAllInstances(isSso);
 
+        apiService.setHideServiceInfo(hideServiceInfo);
```

Comment:
the way how this is done looks like it is configurable per service, but the flag can be specified only in API Catalog. What is the reason for setting this value separately for every service?

Reply:
Yes you're right, I will set it as part of the container, not of the `apiService`, that should be generic